internal/criocli/criocli.go

```diff
@@ -1182,6 +1185,13 @@ func getCrioFlags(defConf *libconfig.Config) []cli.Flag {
 			EnvVars: []string{"DISABLE_HOSTPORT_MAPPING"},
 			Value:   defConf.DisableHostPortMapping,
 		},
+		&cli.StringFlag{
+			Name:    "time-zone",
+			Aliases: []string{"tz"},
+			Usage:   "Specifies the timezone configuration for CRI-O. If not specified, CRI-O will use the host's timezone but displays in UTC.",
```

Comment:
This help is different from the one we quote in the configuration file. The last few phrases of the sentence could be made a little better. :smile: